apps/climatemappedafrica/src/lib/data/blockify/index.js

```diff
@@ -12,13 +12,19 @@ const propsifyBlockBySlug = {
   tutorial,
 };
 
-export const blockify = async (blocks, api, context, hurumap) => {
+export const blockify = async (
+  blocks,
+  api,
+  context,
+  hurumap,
+  hurumapProfile,
```

Comment:
Discuss with the team but I think the signature should be something like:
```js
blockify(page, api, context, settings) {
```

1. from `page`, we get `{ blocks }`,
2. `api` and `context` we all know already,
3. `settings` woulbe be `{ site, hurumap, <any other future setting> }`, and
4. `hurumap` will resemble-ish what we have in the CMS i.e. `{ profile, geography, tutorial, panels }`

Reply:
Another signature option, and probably a better one since it'll align with signature of `getPageProps` is:
```js
blockify(api, context, page, settings) {
```